src/openMVG/sfm/pipelines/sequential/sequential_SfM.hpp

```diff
@@ -34,7 +34,8 @@ class SequentialSfMReconstructionEngine : public ReconstructionEngine
   SequentialSfMReconstructionEngine(
     const SfM_Data & sfm_data,
     const std::string & soutDirectory,
-    const std::string & loggingFile = "");
+    const std::string & loggingFile = "",
+    const openMVG::tracks::STLMAPTracks & map_tracks = {});
```

Comment:
is this ok or should we make a whole new constructor for this ?

Reply:
I think it's dangerous since now we can set the tracks from outside. And so it means that we would like to change the SequentialSfMReconstructionEngine to check if tracks are present before to run the track computation...